recipes/pangeo-dask/LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright 2020 Scott Henderson
```

Comment:
@scottyhq is this OK? Or do you want to add more pangeo folks here?

Reply:
If it's okay to link to the pangeo website here, we should do that https://pangeo.io ... otherwise I think it's ok to leave my name in there

Reply:
I'm not a lawyer to answer that. Let's leave your name for now and you, as the copyright owner ;-p, can update to anything you want in the feedstock later.